api/lib/environment.config.ts

```diff
@@ -41,8 +41,9 @@ const config: EnvironmentConfig = {
     ccdDefApi: getEnvConfig<string>('CCD_DEFINITION_API_SERVICE', 'string'),
     idamApi: getEnvConfig<string>('IDAM_API_SERVICE', 'string'),
     idamWeb: getEnvConfig<string>('IDAM_WEB_SERVICE', 'string'),
+    iss: getEnvConfig<string>('S2S_SERVICE', 'string'),
```

Comment:
just have a check @borislavpetrovhmcts but these values seem the wrong way round? Usings S2S_SERVICE env for the iss, and using ISS env for s2s?